apps/omg_watcher/priv/repo/migrations/20190408131000_add_missing_indices_to_txoutputs.exs

```diff
@@ -0,0 +1,11 @@
+defmodule OMG.Watcher.Repo.Migrations.AddMissingIndicesToTxOuputs do
+  use Ecto.Migration
+
+  def change do
+    create index(:txoutputs, [:creating_txhash, :spending_txhash])
+    create index(:txoutputs, [:creating_deposit])
+    create index(:txoutputs, [:spending_txhash])
```

Comment:
This is @kasima – add an NB about why a `spending_txhash` index alone.

Reply:
Composite indexes will be used by the optimizer when either the first column is used, or both columns are used in the query. However, if a query only uses the second column (`spending_txhash`), the composite index will not be used, hence the addition of another index for it.